Podcast/UserEpisodeData.swift

```diff
@@ -0,0 +1,61 @@
+//
+//  UserEpisodeData.swift
+//  Podcast
+//
+//  Created by Natasha Armbrust on 5/9/18.
+//  Copyright © 2018 Cornell App Development. All rights reserved.
+//
+
+import UIKit
+
+// essentially tuples to store our hashmap of blurbs (swift tuples aren't hashable)
+struct EpisodeToUser: Hashable {
+    let episodeID: String
+    let userID: String
+
+    var hashValue: Int {
+        return "\(episodeID), \(userID)".hashValue
+    }
+
+    static func == (lhs: EpisodeToUser, rhs: EpisodeToUser) -> Bool {
+        return lhs.episodeID == rhs.episodeID && lhs.userID == rhs.userID
+    }
+}
+
+// all info about users to episodes
+class UserEpisodeData: NSObject {
+
+    static var shared: UserEpisodeData = UserEpisodeData()
+
+    var blurbs: [EpisodeToUser: String]
+
+    private override init() {
+        blurbs = [:]
+    }
+
+    func update(with blurb: String?, for episodeToUser: EpisodeToUser) {
```

Comment:
So for example, this above would become: update(with blurb:, for user:  and episode:)

Reply:
Yea actually that might be better so we don't have random models floating around. I can also make blurbs a private variable so it is only ever accessed through the methods in UserEpisodeData

Reply:
To reduce developer mistakes